pystruct/learners/structured_perceptron.py

```diff
@@ -8,6 +8,10 @@ def inference(model, x, w):
     return model.inference(x, w)
 
 
+def inference_map(args):
```

Comment:
I will move all these wrappings into the upstream submodule and just import the wrapped functions as well.